test/e2e.go

```diff
@@ -31,6 +31,15 @@ var _ = Describe("E2E Test", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("Schedule XQueueJob", func() {
+		context := initTestContext()
+		defer cleanupTestContext(context)
+		rep := clusterSize(context, oneCPU)
+		xqueueJob := createXQueueJob(context, "xqj-1", 2, rep, "busybox", oneCPU)
```

Comment:
It seems we only create StatefulSet here, what are we going to test? XQueueJob just a wrapper of StatefulSet?